vmaas/common/config.py

```diff
@@ -97,6 +110,26 @@ def clowder(self) -> None:
 
         self.metrics_port = self._cfg.metricsPort
 
+    def env(self) -> None:
+        """Configuration from environment variables"""
+        self.db_name = os.getenv("POSTGRESQL_DB", "vulnerability").strip()
+        self.db_user = os.getenv("POSTGRESQL_USER", "").strip()
+        self.db_pass = os.getenv("POSTGRESQL_PASSWORD", "")
+        self.db_host = os.getenv("POSTGRESQL_HOST", "ve_database").strip()
+        self.db_port = int(os.getenv("POSTGRESQL_PORT", "5432").strip())
+        self.db_ssl_mode = os.getenv("POSTGRESQL_SSL_MODE", "prefer")
+        self.db_ssl_root_cert_path = os.getenv("POSTGRESQL_SSL_ROOT_CERT_PATH", "")
```

Comment:
what is the reason for some envs having default while others having default as an empty string? Same applies for usage of `.strip()` only for some env variables. Can it be unified please?